app/components/ui/LandingPageResourceBlock.jsx

```diff
@@ -14,11 +15,16 @@ class LandingPageResourceBlock extends Component {
               {this.props.config.TITLE.BEFORE_BLUE_WORD}
               <span className="blue-word">
                 {' '}
+                <a href={url}>
                 {this.props.config.TITLE.BLUE_WORD}
```

Comment:
Nit: You should indent the children of an HTML tag so that the visual indentation makes it easier to see which tags are inside of which other tags.